rero_ils/modules/monitoring.py

```diff
@@ -270,6 +273,43 @@ def missing_pids(doc_type):
         return jsonify({'data': data})
 
 
+@api_blueprint.route('/redis')
+@check_authentication
+def redis():
+    """Displays redis info.
+
+    :return: jsonified redis info.
+    """
+    url = current_app.config.get('ACCOUNTS_SESSION_REDIS_URL',
+                                 'redis://localhost:6379')
+    redis = Redis.from_url(url)
+    info = redis.info()
+    return jsonify({'data': info})
+
+
+@api_blueprint.route('/timestamps')
+@check_authentication
+def timestamps():
+    """Get time stamps from current cache.
```

Comment:
Can you explain a bit more what are timestamps?

Reply:
added:
```
Makes the timestamps accessible via url requests.
```